### ISSUE TITLE
TensorFlow 2: What does "not anytime soon" mean?

### ISSUE THREAD

**Issue by baerbock** [mozilla/DeepSpeech#2709]
As issues on TensorFlow 2 compatibility are quickly getting locked
* https://github.com/mozilla/DeepSpeech/issues/2506
* https://github.com/mozilla/DeepSpeech/issues/2697
I have no other chance than to open a new issue.

 What does "not anytime soon" mean regarding TF2 support?
2021, 2022 or even later?

**Comment by reuben** [comment]
No idea. So far TF2 does not have anything of benefit to DeepSpeech.

**Comment by reuben** [comment]
Maybe let me flip this question back to you: how would TF2 compatibility in DeepSpeech benefit you?

**Comment by baerbock** [comment]
No need to have different versions of TF in parallel.
